### DISCUSSION TITLE
Question about machine.json in PBS cluster

### DISCUSSION THREAD

**Discussion by hyuqing11** [deepmodeling/dpdispatcher#424] [Q&A]
In our PBS cluster, we cannot specify the queue name when submitting jobs. The router queue will send the jobs to the right place.  However, when utilizing the machine.json configuration, it automatically generates #PBS -q by default. I am seeking suggestions on how to remove or modify this default setting.

**Comment by njzjz** [comment]
See https://github.com/deepmodeling/dpdispatcher/issues/406#issuecomment-1789933518

**Reply by hyuqing11** [reply]
Thank you for your help. It works now.
